apps/autoscheduler/AutoSchedule.h

```diff
@@ -0,0 +1,19 @@
+#include "Halide.h"
+#include "CostModel.h"
+#include "FunctionDAG.h"
+#include "PerfectHashMap.h"
+#include "Featurization.h"
+#include <vector>
+
+namespace Halide {
+namespace Internal {
+
+typedef PerfectHashMap<FunctionDAG::Node::Stage, ScheduleFeatures> StageMapOfScheduleFeatures;
+
+extern "C" {
```

Comment:
Why extern "C" linkage?

Reply:
This was a quick way to work around the fact that the FunctionDAG class was in an anonymous namespace, and couldn't be linked properly. This isn't required anymore.